app/views/providers/open_banking_guidances/show.html.erb

```diff
@@ -7,14 +7,15 @@
 
   <%= page_template page_title: t(".heading_h1"), form:, template: :basic do %>
 
+    <span class="govuk-caption-l"><%= t("generic.client_means_caption") %></span>
```

Comment:
Discussion: the design pattern generally as both title and caption at the same size, what are our reasons for differing from this pattern?